File: Modul-5/Soal 1/PRAK501-2410817110009-MuhammadAlfiGunawan.py
def MaxBilangan(w,x,y,z):
    if w > x and w > y and w > z:
        besar = w
    elif x > w and x > y and x > z:
        besar = x
    elif y > w and y > x and y > z:
        besar = y
    elif z > w and z > x and z > y:
        besar = z
    return besar

a,b,c,d = map(int,input().split())
hasil = MaxBilangan(a,b,c,d)
print(hasil)

File: Modul-5/Soal 1/PRAK501-2410817110009-MuhammadAlfiGunawan.c
#include <stdio.h>
int MaxBilangan(int w, int x, int y, int z){
    int besar;
    if (w > x && w > y && w > z){
        besar = w;
    }
    else if (x > w && x > y && x > z){
        besar = x;
    }
    else if (y > w && y > x && y > z){
        besar = y;
    }
    else {
        besar = z;
    }
    return besar;
}
int main() {

    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);

    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
return 0;
}
